app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,24 @@
+class SessionsController < ApplicationController
+	def new
+	end
+	
+	def create
+		user = User.find_by(email: params[:session][:email])
+
+		if user && user.authenticate(params[:session][:password])
+			session[:user_id] = user.id
```

Comment:
@saki4567 
Change this sentence to:
`log_in user`
, which is related to this comment: 
https://github.com/framgia831/rails-elearning-haruna/pull/2/files#diff-c70c347141b21b6f03e13ad752ecca26R5